### ISSUE TITLE
[2016.11] 2 boto_vpc test failures

### ISSUE THREAD

**Issue by rallytime** [saltstack/salt-jenkins#692]
One is a module test failure:
- unittest.loader.ModuleImportFailure.unit.modules.boto_vpc_test
```
ImportError: Failed to import test module: unit.modules.boto_vpc_test
Traceback (most recent call last):
  File "/usr/lib/python2.7/unittest/loader.py", line 254, in _find_tests
    module = self._get_module_from_name(name)
  File "/usr/lib/python2.7/unittest/loader.py", line 232, in _get_module_from_name
    __import__(name)
  File "/testing/tests/unit/modules/boto_vpc_test.py", line 134, in <module>
    @skipIf(_has_required_moto() is False, 'The moto version must be >= to version {0}'.format(required_moto_version))
  File "/testing/tests/unit/modules/boto_vpc_test.py", line 120, in _has_required_moto
    if _get_moto_version() < LooseVersion(required_moto_version):
  File "/testing/tests/unit/modules/boto_vpc_test.py", line 104, in _get_moto_version
    return LooseVersion(moto.__version__)
  File "/usr/lib/python2.7/distutils/version.py", line 265, in __init__
    self.parse(vstring)
  File "/usr/lib/python2.7/distutils/version.py", line 274, in parse
    self.component_re.split(vstring))
TypeError: expected string or buffer
```

The other is a state test failure:
- unittest.loader.ModuleImportFailure.unit.states.boto_vpc_test
```
ImportError: Failed to import test module: unit.states.boto_vpc_test
Traceback (most recent call last):
  File "/usr/lib/python2.7/unittest/loader.py", line 254, in _find_tests
    module = self._get_module_from_name(name)
  File "/usr/lib/python2.7/unittest/loader.py", line 232, in _get_module_from_name
    __import__(name)
  File "/testing/tests/unit/states/boto_vpc_test.py", line 24, in <module>
    from unit.modules.boto_vpc_test import BotoVpcTestCaseMixin
  File "/testing/tests/unit/modules/boto_vpc_test.py", line 134, in <module>
    @skipIf(_has_required_moto() is False, 'The moto version must be >= to version {0}'.format(required_moto_version))
  File "/testing/tests/unit/modules/boto_vpc_test.py", line 120, in _has_required_moto
    if _get_moto_version() < LooseVersion(required_moto_version):
  File "/testing/tests/unit/modules/boto_vpc_test.py", line 104, in _get_moto_version
    return LooseVersion(moto.__version__)
  File "/usr/lib/python2.7/distutils/version.py", line 265, in __init__
    self.parse(vstring)
  File "/usr/lib/python2.7/distutils/version.py", line 274, in parse
    self.component_re.split(vstring))
TypeError: expected string or buffer
```

This is failing at the HEAD of 2016.11. Currently seeing it also on PRs, so this is likely due to a package upgrade or something.

**Comment by gtmanfred** [comment]
the moto.__version__ object is now a tuple instead of a string, part of the release yesterday https://github.com/spulec/moto/releases/tag/1.2.0

**Comment by rallytime** [comment]
Let's just pin this package. It causes more work than it should.

Fix is in #697
